site/content/docs/main/sonobuoy-query.md

```diff
@@ -0,0 +1,18 @@
+# New `sonobuoy query` command
```

Comment:
Since we are putting this in docs and not a blog, don't refer to this as 'new'. Just make the title `Sonobuoy Query Command`